src/poliastro/tests/test_czml.py

```diff
@@ -154,8 +169,8 @@ def test_czml_add_orbit():
         ]
     },
     "billboard": {
-        "show": true,
```

Comment:
What version did you use? Notice that we are pinning czml3 to be 0.4.0 in `pyproject.toml`, but there's a more recent one with a change

Reply:
I didn't use conda to install anything. I was following the coding guidelines. One of the steps was to install flit and it was adding and removing packages. I don't know this tool, so don't understand what was happening. pip claims czml3 is 0.4.0.

Reply:
I see! the order change here shouldn't be necessary then, as far as I understand, right? or were these tests failing too?

Reply:
The czml_add_orbit test failed with and without this change. Without this change, there was additional error complaining about the difference:

```
E               "billboard": {
E         -         "show": true,
E         -         "image": "data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAYAAAAf8/9hAAAAAXNSR0IArs4c6QAAAARnQU1BAACxjwv8YQUAAAAJcEhZcwAADsMAAA7DAcdvqGQAAADJSURBVDhPnZHRDcMgEEMZjVEYpaNklIzSEfLfD4qNnXAJSFWfhO7w2Zc0Tf9QG2rXrEzSUeZLOGm47WoH95x3Hl3jEgilvDgsOQUTqsNl68ezEwn1vae6lceSEEYvvWNT/Rxc4CXQNGadho1NXoJ+9iaqc2xi2xbt23PJCDIB6TQjOC6Bho/sDy3fBQT8PrVhibU7yBFcEPaRxOoeTwbwByCOYf9VGp1BYI1BA+EeHhmfzKbBoJEQwn1yzUZtyspIQUha85MpkNIXB7GizqDEECsAAAAASUVORK5CYII="
E         +         "image": "data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAYAAAAf8/9hAAAAAXNSR0IArs4c6QAAAARnQU1BAACxjwv8YQUAAAAJcEhZcwAADsMAAA7DAcdvqGQAAADJSURBVDhPnZHRDcMgEEMZjVEYpaNklIzSEfLfD4qNnXAJSFWfhO7w2Zc0Tf9QG2rXrEzSUeZLOGm47WoH95x3Hl3jEgilvDgsOQUTqsNl68ezEwn1vae6lceSEEYvvWNT/Rxc4CXQNGadho1NXoJ+9iaqc2xi2xbt23PJCDIB6TQjOC6Bho/sDy3fBQT8PrVhibU7yBFcEPaRxOoeTwbwByCOYf9VGp1BYI1BA+EeHhmfzKbBoJEQwn1yzUZtyspIQUha85MpkNIXB7GizqDEECsAAAAASUVORK5CYII=",
E         ?                                                                                                                                                                                                                                                                                                                                                                                                                                                                      +
E         +         "show": true
E               },
```

I can revert this and the other change that changed the show/image order if you want.

Reply:
Oh sorry, this was the test that lacked the `assert`. Let's keep it this way for now!